Tools/LyTestTools/ly_test_tools/o3de/editor_test.py

```diff
@@ -784,7 +784,7 @@ def _exec_editor_test(self, request: Request, workspace: AbstractWorkspace, edit
                     test_result = Result.Fail.create(test_spec, output, editor_log_content)
         except WaitTimeoutError:
             output = editor.get_output()
-            editor.kill()
+            editor.stop()
```

Comment:
Have you checked the timing difference for the tests doing before vs. after this change?
I think its really possible that using this approach will have a big toll on the total test times